che-theia-java-extension/src/browser/navigation/file-structure.ts

```diff
@@ -229,25 +227,25 @@ export class MemberQuickOpenItem extends QuickOpenItem {
     }
 
     getDescription(): string {
-        return "";
+        return '';
     }
 
     getIconClass() {
         switch (this.kind) {
             case SymbolKind.Interface: {
-                return "java-interface-icon file-icon";
+                return 'java-interface-icon file-icon';
```

Comment:
I would much prefer it if you left cleanup task that have nothing to do with the issue you're working on for separate PR's. It just makes it harder to review. Also, if I want to reject your cleanup, you'll have to rework this PR, if you had the cleanup in one PR, that would be simpler.